packages/pharos/src/components/image-card/pharos-image-card.wc.stories.jsx

```diff
@@ -47,12 +52,7 @@ export const Base = {
       ${items.map((item, index) => {
         return html` <li style="grid-column: span 2">
           <storybook-pharos-image-card id="card-${index}" title="Card Title" link="#">
-            <img
-              id="image-${index}"
-              src="./images/item-detail/${item.image}"
-              alt="Card Title ${index}"
-              slot="image"
-            />
+            <img id="image-${index}" src="${item.image}" alt="Card Title ${index}" slot="image" />
```

Comment:
Tangential to this PR, is `index` being injected into these strings correctly? I thought that using back-ticks was necessary for the `${}` template syntax to work.

Reply:
Note the backticks several lines up where the `html` tagged literal opens!